dashboard/config.ru

```diff
@@ -17,8 +17,15 @@ unless rack_env?(:development)
     }
 end
 
-require 'gctools/oobgc/unicorn_middleware'
-use GC::OOB::UnicornMiddleware
+# Temporarily wrap this unicorn-specific middleware in a DCDO flag so we can
+# evaluate whether or not this still has a performance impact now that we're no
+# longer using unicorn.
+# TODO: either remove the flag or this entire block, depending on the results
+unless DCDO.get('oobgc_unicorn_middleware_disabled', false)
```

Comment:
Do we typically use Gatekeeper for boolean (“feature flag”) decisions, and DCDO more for configuration settings that have numeric or string datatypes? This should work either way.

Reply:
We typically use Gatekeeper for the [more advanced query logic it provides](https://github.com/code-dot-org/code-dot-org/blob/704c0fb3908c3dc4d5c48aafb0fb4f379e6509e3/lib/dynamic_config/gatekeeper.rb#L11-L15), which allows us to conditionally enable or disable things based on request parameters. It's also generally a lot more complex; DCDO is the simple option, and is more applicable for these kind of global config options